src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectIssueResource.java

```diff
@@ -0,0 +1,200 @@
+		List<Issue> closedIssues = issues.findClosedIssues(repositoryEntity);
+
+		Map<String, Object> parameters = getBaseParameters();
+		
+		parameters.put("repository", repository);
+		parameters.put("openIssues", openIssues);
+		parameters.put("closedIssues", closedIssues);
+
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issues.ftl", locales, parameters));
+	
+	}
+	
+	@GET
+	@Transactional
+	@Path("/issues/create")
+	public Response openCreateIssuePage(@Context HttpServletRequest request) throws IOException{
+		
+		RepositoryEntity repositoryEntity = getRepositoryEntity();
+		RepositoryApi repositoryApi = getRepositoryApi(repositoryEntity);
+		RepositoryModel repository = repositoryApi.getRepositoryModel();
+		
+		Map<String, Object> parameters = getBaseParameters();		
+		parameters.put("repository", repository);
+		
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issue-create.ftl", locales, parameters));
+	}
+
+	@POST
+	@Transactional
+	@Path("/issues/create")
+	public Response createIssue(@Context HttpServletRequest request,
+			@FormParam("title") String title,
+			@FormParam("description") String description,
+			@FormParam("assignee") String assigneeNetID) throws IOException{
+		
+		Issue issue = new Issue();
+		
+		issue.setTitle(title);
+		issue.setDescription(description);
+		issue.setOpen(true);
+		
+		User assignee = users.findByNetId(assigneeNetID);
+		checkCollaborator(assignee);
+		
+		issue.setAssignee(assignee);
+		issue.setTimestamp(new Date());
+		issue.setRepository(getRepositoryEntity());
+		
+		issueBackend.createIssue(getRepositoryApi(getRepositoryEntity()), issue);
+		
+		return redirect(issue.getURI().toString());
+	}
+	@GET
+	@Transactional
+	@Path("/issue/{issueId}")
+	public Response viewIssue(@Context HttpServletRequest request, 
+			@PathParam("issueId") long issueId) throws IOException {
+
+		RepositoryEntity repositoryEntity = getRepositoryEntity();
+		RepositoryApi repositoryApi = getRepositoryApi(repositoryEntity);
+		RepositoryModel repository = repositoryApi.getRepositoryModel();
+		
+		Issue issue = issues.findIssueById(getRepositoryEntity(), issueId).get(0);
+		
+		Map<String, Object> parameters = getBaseParameters();		
+		parameters.put("repository", repository);		
+		parameters.put("issue", issue);
+		
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issue-create.ftl", locales, parameters));
+	}
+
+	@POST
+	@Transactional
+	@Path("/issue/{issueId}")
+	public Response updateIssue(@Context HttpServletRequest request, 
+			@PathParam("issueId") long issueId,
+			@FormParam("title") String title,
+			@FormParam("description") String description,
+			@FormParam("assignee") String assigneeNetID,
+			@FormParam("status") String status) throws IOException {
```

Comment:
I believe you can even use a `Boolean` here, which has the same type as the field on `PullRequest`, where `=true` is `=true`, `false` is `false` and undefined is `null`. If you create the enum instead  I would specifically make one for the resource as this is API-wise convenience, and changing it in the entity as well will be very complicated, as you have to change the field, its Hibernate mapping, and probably some sophisticated database changelog as well :wink: